### PR TITLE
add: ctwm (attempt 2)

### LINKED ISSUE #1030: [Request] CTWM
**What software are you requesting to be packaged?**
CTWM https://www.ctwm.org/index.html

**Describe the motivation**

Funny window manager for this set of dotfiles
http://scarlettppcblog.ddns.net/guides/ctwmpinkv1.html

**Additional context**
It's in RPMSphere already?

### PR DESCRIPTION
**Pull request by nothingneko**
closes #1030

### CODE REVIEW

File: anda/desktops/ctwm/anda.hcl [hunk 1]
```diff
@@ -0,0 +1,5 @@
+  project pkg {
```
Comment (madonuko) [inline anda/desktops/ctwm/anda.hcl]: indent

File: anda/desktops/ctwm/ctwm.spec [hunk 2]
```diff
@@ -0,0 +1,54 @@
+Summary: Lightweight window manager with virtual workspaces
+Name: ctwm
+Version: 4.1.0
```
Comment (madonuko) [inline anda/desktops/ctwm/ctwm.spec:3]:
```ts
rpm.version(find("<p>Current release: ([\\d.]+)</p>", get("https://www.ctwm.org/download.html"), 1));
```

File: anda/desktops/ctwm/ctwm.spec [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+Release: 1
```
Comment (madonuko) [inline anda/desktops/ctwm/ctwm.spec]:
```suggestion
Release: 1%?dist
```

File: anda/desktops/ctwm/ctwm.spec [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+%cmake
+%cmake_build
+
+%install
+%cmake_install
+%{__install} -Dm644 %{SOURCE1} $RPM_BUILD_ROOT%{_datadir}/xsessions/%{name}.desktop
+
+%files
+%doc README.md CHANGES.md
+%doc README.md CHANGES.md
```
Comment (madonuko) [inline anda/desktops/ctwm/ctwm.spec]:
```suggestion
```

**Review by madonuko — CHANGES_REQUESTED** [review]

File: anda/desktops/ctwm/ctwm.spec [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+%license COPYRIGHT
+%{_bindir}/%{name}
+%{_mandir}/man1/%{name}*
+%{_datadir}/xsessions/%{name}.desktop
+%{_datadir}/ctwm
+%{_datadir}/doc/ctwm/ctwm.1.html
+%{_datadir}/examples/ctwm/system.ctwmrc
+
+%changelog
+* Thu Jun 27 2024 Jaiden Riordan <jade@fyralabs.com> - 4.1.0
```
Comment (nothingneko) [inline anda/desktops/ctwm/ctwm.spec:40]: @Owen-sz  take some credit

Comment (Owen-sz) [reply anda/desktops/ctwm/ctwm.spec:40]: Ok lol

**Comment by Owen-sz** [comment]
Mado please verify the rhai/sources are good (referring to thishttps://github.com/terrapkg/packages/pull/1039/files#r1550841803)

**Review by madonuko — APPROVED** [review]

**Comment by raboneko** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|f39|[](https://github.com/terrapkg/packages/pull/2671)|
|✅|f40|[](https://github.com/terrapkg/packages/pull/2672)|
|✅|f41|[](https://github.com/terrapkg/packages/pull/2673)|
|✅|el10|[](https://github.com/terrapkg/packages/pull/2674)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport) and see the [Github Action logs](https://github.com/terrapkg/packages/actions/runs/12398388727) for details

**Pull request merged**